pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -79,7 +79,7 @@ const (
 	// ndots specifies the minimum number of dots that a domain name must contain for the resolver to consider it as FQDN (fully-qualified)
 	// we want to able to consider SRV lookup names like _dns._udp.kube-dns.default.svc to be considered relative.
 	// hence, setting ndots to be 5.
-	ndotsDNSOption = "options ndots:5\n"
+	ndotsDNSOption = "options use-vc ndots:5\n"
```

Comment:
I see you already mention this in your PR comments - apologies for the duplication :-)